config/config.json

```diff
@@ -1,10 +1,10 @@
 {
   "development": {
     "username": "root",
-    "password": "secret",
```

Comment:
I'll add another set of config variables for my machine :P 
I suggest the same for our members